okio/src/appleMain/kotlin/okio/posixVariant.kt

```diff
@@ -25,8 +25,8 @@ import platform.posix.S_IFREG
 import platform.posix.errno
 import platform.posix.stat
 
-@ExperimentalFilesystem
-internal actual fun PosixSystemFilesystem.variantMetadataOrNull(path: Path): FileMetadata? {
+@ExperimentalFileSystem
+internal actual fun PosixSystemFileSystem.variantMetadataOrNull(path: Path): FileMetadata? {
```

Comment:
nit: any reason this isn't just `PosixFileSystem`?

Reply:
Lemme make that change!